CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 2.0.2 (WIP)
+- Upgrade dependencies to resolve trivy scan failures.
```

Comment:
Should we also add - logs issue if proxy is not set via environment? For your fix below?